.eslintrc

```diff
@@ -31,12 +30,20 @@
   },
   "overrides": [
     {
-      "files": ["__tests__/__fixtures__/*.js", "src/cli/*.js", "src/cli/commands/*.js", "src/cli/lib/*.js"],
+      "files": ["*.js"],
```

Comment:
will this lead to node_modules being linted? Is there a way to do a verbose run and output all the files we're checking to make sure nothing obvious wrong is now included?

Reply:
ESLint ignores `node_modules/` by default, but running it with `--debug` it doesn't look in there still.